src/mongoshake/common/common.go

```diff
@@ -19,7 +19,7 @@ const (
 )
 
 // Build info
-var VERSION = "$"
+var BRANCH = "$"
```

Comment:
could you plz modify all VERSION to BRANCH in all code?

Reply:
Modify all VERSION to BRANCH in https://github.com/aliyun/mongo-shake/pull/21 .